test/fresh_service/api_test.exs

```diff
@@ -0,0 +1,63 @@
+defmodule FreshService.ApiTest do
```

Comment:
I saw doctests in `FreshService.Api` module. We should run them here.

Reply:
Nice, didn't know about the doctests ;)